types/kurento-client/kurento-client-tests.ts

```diff
@@ -56,6 +56,15 @@ async () => {
     return { cpuUsage: server.getUsedCpu(5) };
 };
 
+async () => {
+    const endpointId = 'endpointId';
+
+    const client = await kurento('//server', { failAfter: 500, useImplicitTransactions: true });
+    const endpoint = await client.getMediaobjectById<WebRtcEndpoint>(endpointId);
```

Comment:
```suggestion
    const endpoint = await client.getMediaobjectById<WebRtcEndpoint>(endpointId); // $ExpectType WebRtcEndpoint
```

I think it would be better if type assertion is added.